Gruntfile.js

```diff
@@ -469,6 +475,48 @@ module.exports = function (grunt) {
                     compress: true
                 }
             }
+        },
+        verifyDistFiles: {
+            js: {
+                verifyFilesListFunc: function () {
+                    var expectedFilenames = [];
+                    var distributions = grunt.config.get("distributions");
+                    _.forEach(distributions, function (value, distribution) {
+                        var jsFilename = "infusion-" + distribution + ".js";
+                        var mapFilename = jsFilename + ".map";
+                        var distDirectory = "dist/";
+                        expectedFilenames.push(distDirectory + jsFilename, distDirectory + mapFilename);
+                    });
+                    return expectedFilenames;
```

Comment:
Similar to previous comment, can use the map method instead of forEach.

Reply:
May want to consider defining this function outside of the config and just passing in the reference to it here.